drivers/dma/Kconfig.stm32

```diff
@@ -43,3 +43,21 @@ config DMA_STM32_SHARED_IRQS
 	  Enable shared IRQ support on devices where channels share 1 IRQ.
 
 endif # DMA_STM32
+
+
+config BDMA_STM32
+	bool "Enable STM32 BDMA driver"
+	select USE_STM32_LL_BDMA
+	depends on SOC_FAMILY_STM32
+	help
+	  DMA driver for STM32 series SoCs.
+
+if BDMA_STM32
+
+config DMAMUX_STM32
+	bool
+	default $(dt_compat_enabled,$(DT_COMPAT_ST_STM32_DMAMUX))
+	help
+	  Enable DMAMUX support.
+
+endif # BDMA_STM32
```

Comment:
This is a little confusing, because there is no two entries for `DMAMUX_STM32`. I think we should consolidate it to one
with something like
```kconfig
if DMA_STM32 || BDMA_STM32

config DMAMUX_STM32
	bool
	default $(dt_compat_enabled,$(DT_COMPAT_ST_STM32_DMAMUX))
	help
	  Enable DMAMUX support.

endif # BDMA_STM32
```
You could probably put your name at the top of the file too :P

Reply:
Done